README.rst

```diff
@@ -127,38 +127,95 @@ These files should be located at ``/edx/app/edxapp/`` directory, see the example
     }
 
 2. Add the following keys and their values in the ``lms.env.json`` and ``cms.env.json`` files.
+Please note that all parameters in the `PRODUCER_CONFIG` are unique to the broker instances. You
+can set whatever parameters are required for you.
```

Comment:
nit:
```suggestion
can set whatever parameters required for your instance.
```

Reply:
Updated 'you' with 'your instance' but don't think 'are' should be removed.